module1.js

```diff
@@ -28,7 +29,11 @@ function question1(name1) {
 */
 
 function question2(number1, number2) {
-  // TODO:
+  if (number1 == number2 || number2 == number1) {
```

Comment:
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons This is a good read about using `==` in JavaScript.

Also, if you're comparing two things, why do you need to compare twice? Say, if you're comparing `apples` and `oranges`, is it not enough to ask "Are apples the same as oranges"? You don't have to ask again "Are oranges the same as apples?" The outcome is always the same. Ask once :)

Reply:
Also also, I just realised you used `===` elsewhere, so why did you decide to use `==` here?

Reply:
I wrote only == instead of 3 thinking in case the number could be also a string, but reading the exercise it was only integers so it should be ===.

Reply:
Fair. I'd maybe ask Jon or Ali about it (or whoever's in the class tomorrow) but I'd say it's safer to use `===` because it won't try to convert stuff for you. For example, you might get a nasty error if you try to compare a number and a string like `0 == 'apple'` not sure what would happen then, but `0 === 'apple'` would definitely just return false.